config.py

```diff
@@ -2,11 +2,16 @@
 import socket
 
 app_name = os.getenv('HEROKU_APP_NAME', socket.gethostname())
-dyno = os.getenv('DYNO')
+dyno = os.getenv('DYNO')  # if this env variable is set, it means we are in a Heroku
 athena_query = os.getenv('AWS_ATHENA_PREPARE_STATEMENT')
 
 config = {
-    'port': os.getenv('PORT') or 8080,
+    'port': os.getenv('PORT', 8080),
+    # I can't reference a previous field, so copying and pasting here
```

Comment:
Yes, if you see the definition of the `domain_name` field there's a part where I write `os.getenv('PORT', 8080)` again, this is because I can't do `config['port']` since we are just defining the dictionary.